mlir/include/mlir/Dialect/Transform/IR/TransformAttrs.td

```diff
@@ -33,4 +41,17 @@ def MatchCmpIPredicateAttr : I32EnumAttr<
   let cppNamespace = "::mlir::transform";
 }
 
+def ParamOperandIndexAttr : Transform_Attr<"ParamOperandIndex",
+                                           "param_operand_index" > {
+  let mnemonic = "param_operand_index";
```

Comment:
nit: I'd skip `_index` to make read like you refer directly to a param

Reply:
Moved `index` inside the argument list, as a keyword before the argument.